js/views/ListingCard.js

```diff
@@ -405,6 +405,8 @@ export default class extends baseVw {
       this.listenTo(this.reportBtn, 'startReport', this.startReport);
       this.getCachedEl('.js-reportBtnWrapper').append(this.reportBtn.render().el);
     }
+    // hide unneeded report buttons so their wrapper doesn't affect the button layout
+    this.getCachedEl('.js-reportBtnWrapper').toggleClass('hide', !this.reportsUrl);
```

Comment:
Would it be better to just style the wrapper to hide itself when't empty? `.wrapper:empty { display: none }`

The advantage to that is if you ever need to fill it based on some event, it would be ready for you (even though that's probably not needed in this case). Also saves a dom write.

We could even create a generic `hideIfEmpty` class, since this pattern is used a time or two in the code already.